tsl/test/sql/continuous_aggs_permissions.sql

```diff
@@ -8,6 +8,28 @@
 -- remove any default jobs, e.g., telemetry so bgw_job isn't polluted
 DELETE FROM _timescaledb_config.bgw_job WHERE TRUE;
 
+CREATE VIEW cagg_info AS
+WITH
+  caggs AS (
+    SELECT format('%s.%s', user_view_schema, user_view_name)::regclass AS user_view,
+           format('%s.%s', direct_view_schema, direct_view_name)::regclass AS direct_view,
+           format('%s.%s', partial_view_schema, partial_view_name)::regclass AS partial_view,
+           format('%s.%s', ht.schema_name, ht.table_name)::regclass AS mat_relid
```

Comment:
this format string is not correct, correct format string for all those cases is '%I.%I'

Reply:
Changed.